client/app/components/DateTimeInput.jsx

```diff
@@ -17,23 +17,14 @@ export function DateTimeInput({
   if (value && value.isValid()) {
     additionalAttributes.defaultValue = value;
   }
-  const currentValueRef = useRef(additionalAttributes.defaultValue);
   return (
     <DatePicker
       className={className}
       showTime
       {...additionalAttributes}
       format={format}
       placeholder="Select Date and Time"
-      onChange={(newValue) => { currentValueRef.current = newValue; }}
-      onOpenChange={(status) => {
-        const currentValue = currentValueRef.current;
-        if (!status) { // on close picker
-          if (currentValue && currentValue.isValid()) {
-            onSelect(currentValue);
```

Comment:
@ranbena this logic was used to trigger changes only when the DatePicker was closed, with nis implementation is no longer needed. This was the cause for the clear button issue.